index.html

```diff
@@ -20,25 +21,113 @@
      </nav>
     </header>
      <section class="section">
-        <img src="background.png" alt="" >
+        <img src="background.png" alt="">
 </section>
-    <!-- <section class="my-works">
-        <h1 class="my-works-header">My Projects</h1>
-        <hr>
-        <div class="show-case">
-    <div class="p1"></div>
-    <div class="p2"></div>
-    <div class="p3"></div>
-    <div class="p4"></div>
-    <div class="p5"></div>
-    <div class="p6"></div>
-    <div class="p7"></div>
-    
-    </div>
+</section>
+<div class="works-section">
+    <section>
+    <header>
+        <h1 class="my-works-header"><img src="Title.png" alt=""></h1> 
+        <h2 class="my-works-header2"><img src="Vectorline.png" alt="" width="315"></h2> 
+        </header>
+        <!-- this container is a flex container containing all the cards used. -->
+    <section class="show-case">
+        <img src="Frame44.png" alt="">
+   <div class="project-list1">
+    <div class="p-image"></div>
+   </div>
+   <!-- //this container list all projects on the site -->
+   <div class="project-list">
+<section>
+<header class="P-Info1"><h1></h1></header>
+<p class="content-1"></p>
+<ul>
+    <li class="css"></li>
+    <li class="html"></li>
+    <li  class="bootstrap"></li>
+    <li  class="ruby"></li>
+</ul>
+<ul><li><button class="se"></button></li></ul>
+
+</section>
+   </div>
+   <div class="project-list2">
+    <section class="t-post">
     
-        </section> -->
+        <header class="P-Info2"><h1 class="t-head2"></h1></header>
+<p class="content-2"></p>
+    <ul class="ul2">
+        <li class="html2"></li>
+        <li  class="bootstrap2"></li>
+        <li  class="ruby2"></li>
+    </ul>
     
-    <!-- <div><p class="side-logo"><i class="fa fa-github" aria-hidden="true"></i><i class="fab fa-linkedin"></i><i class="fab fa-angellist"></i><i class="fa fa-twitter" aria-hidden="true"></i><img src="Icon - Medium"> -->
     </section>
+    <ul><li><button class="se"></button></li></ul>
+   </div>
+    </section>
+<!-- /// this section is the about myself section containing all the information about the owner of the site and his achievenebts -->
+    <section class="About-us">
+      
+    <header>
+        <h1 class="about-us-header"><img src="aboutus.png" alt=""></h1> 
+        </header>
+        <div>
+        <p class="describe"><img src="Description.png" alt="" width="315"></p> 
+        <p class="Get-my-resume"><img src="Enable.png" alt=""></p>
+    </div>
+        <div class="white-image">
+            <img src="Frame.png" alt="">   
```

Comment:
- [ ] I kindly suggest that you do not make use of images for creating your full website instead use HTML markup 
- [ ] I kindly suggest that you use HTML markup and css to create your website and not just paste a screenshot on your webpage.